games/o.yaml

```
@@ -1554,7 +1554,8 @@
   repo: https://github.com/OpenHV/OpenHV
   type: similar
   url: https://openhv.net
-  updated: 2023-03-05
+  added: 2021-01-07
```

```suggestion
  added: 2020-12-30
```
in https://github.com/opengaming/osgameclones/commit/f6c62bceae91541fae988f1cd04ef97a9a04858e